src/ytdl_sub/prebuilt_presets/helpers/media_quality.yaml

```diff
@@ -0,0 +1,40 @@
+presets:
+
+  best_video_quality:
+    format: "bestvideo+bestaudio/best"
+    ytdl_options:
+      merge_output_format: "mp4"
+
+  "Best AV Quality":
```

Comment:
Can we call this 'Best Video Quality', I feel like many people won't know what AV is. That and getting best audio quality seems kinda implicit with getting best video